### DISCUSSION TITLE
Does mdxjs support ruby tag

### DISCUSSION THREAD

**Discussion by tienlx97** [mdx-js/mdx#2191] [Q&A] [answered]
Does mdxjs support ruby tag, ex: 
紳士 -> へん たい

**Comment by ChristianMurphy** [comment, marked as answer]
> Does mdxjs support ruby tag

Yes, using JSX.
Taking the example from MDN https://developer.mozilla.org/en-US/docs/Web/HTML/Element/rt
```html
<ruby>
漢 <rp>(</rp><rt>kan</rt><rp>)</rp>
字 <rp>(</rp><rt>ji</rt><rp>)</rp>
</ruby>
```

and pasting it into the MDX playground https://mdxjs.com/playground/
It works fine.

---

Neither [commonmark](https://commonmark.org/) nor [GFM](https://github.github.com/gfm/) have shorthand syntax for ruby text.
There has been some discussion of adding some here: https://talk.commonmark.org/t/proper-ruby-text-rb-syntax-support-in-markdown/2279 but it has not reached consensus.
You are welcome to contribute to the discussion, and if/when it is standardized in commonmark, remark and mdx would add it.

**Reply by tienlx97** [reply]
tks, I think I should use JSX component
